pages/advanced/index.js

```diff
@@ -5,7 +5,11 @@ import PageSections from '../../components/page-sections'
 
 function Advanced({url, content, locale}) {
   return (
-    <Layout pathname={url ? url.pathname : ''} locale={locale}>
+    <Layout
+      pathname={url ? url.pathname : ''}
+      locale={locale}
+      content={content}
```

Comment:
With the change mentioned above you could change this to: `contributors={content.contributors}`